src/Routes.js

```diff
@@ -2,9 +2,13 @@ import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 import { PrivateRoute, SignInRoute } from "./utils/CustomRoute";
 import SignIn from "./pages/Login/Signin";
 import Friends from "./pages/Friends/friends";
```

Comment:
We should probably change the name of this to be more descriptive, maybe "SearchFriends" or "FindFriends"?